File: php/exercicios/listaligada/Elemento.php
<?php

class Elemento {
    private $valor;
    private Elemento $proximo;

    public function __cosntruct($valor)
    {
        $this->valor = $valor;
    }

    public function __getProximo(): Elemento
    {
        return $this->proximo;
    }

    public function __setProximo($proximo): void
    {
        $this->proximo = $proximo;
    }
    public function __getValor()
    {
        return $this->valor;
    }

    public function __setValor($valor): void
    {
        $this->valor = $valor;
    }
}

File: src/exercicios/lista_ligada/Elemento.java
package exercicios.lista_ligada;

public class Elemento {
    private Elemento proximo;
    private String valor;


    
    public Elemento(String valor) {
        this.valor = valor;
    }

    public Elemento getProximo() {
        return proximo;
    }

    public void setProximo(Elemento proximo) {
        this.proximo = proximo;
    }

    public String getValor() {
        return valor;
    }
    
    public void setValor(String valor) {
        this.valor = valor;
    }

    
}
